kroxylicious-systemtests/pom.xml

```diff
@@ -132,6 +142,18 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-annotations</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.kroxylicious</groupId>
+            <artifactId>kroxylicious-runtime</artifactId>
```

Comment:
I'm surprise to see dependency is needed.

Reply:
I'm using the `FilterDefinitionBuilder` to create the configMap for Kroxylicious and indirectly it calls to `FilterDefinition` that is located in `kroxylicious-runtime`